src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx

```diff
@@ -32,6 +33,20 @@ class EventsTableRow extends React.Component {
     }
   };
 
+  renderMinidumpDownloadLink() {
+    let {orgId, event, projectId} = this.props;
+    if (!event.minidump) {
+      return null;
+    }
+    let url = `/api/0/projects/${orgId}/${projectId}/events/${event.id}/attachments/${event
+      .minidump.id}/?download=1`;
+    return (
+      <small>
+        Minidump: <a href={url}>{event.minidump.name}</a> (<FileSize bytes={event.minidump.size} />)
```

Comment:
Is minidump label here needed? (I could go either way)

```suggestion
        <a href={url}>{event.minidump.name}</a> (<FileSize bytes={event.minidump.size} />)
```

Reply:
I think it’s needed because one can’t guess from the filename